packages/lib/src/vis/utils.ts

```diff
@@ -46,16 +66,27 @@ const adaptedLogTicksThreshold = scaleLinear({
   range: [0.8, 1.4],
 });
 
-export function createAxisScale(
+export function createScale(
   scaleType: VisScaleType,
-  config: VisxScaleConfig | ScaleGammaConfig
-): AxisScale {
-  if (isScaleType(scaleType)) {
-    return H5WEB_SCALES[scaleType].createScale(config);
+  config: ScaleConfig
+): Scale {
```

Comment:
As mentioned, there's definitely an opportunity here to use a generic to infer a stricter return type. This would allow `abscissaScale` and `ordinateScale` to have `type AxisScale =  PickD3Scale<AxisScaleType, number>` instead of just `Scale` (which includes gamma/sqrt). However, I preferred to keep this for a future PR.